### PR TITLE
Add RC 110 Scene button support

### PR DESCRIPTION
**Pull request by mrsteakhouse**
Hi there,

i implemented `moveToLevelWithOnOff` to handle commands sent from scene buttons on a Innr RC 110.
Additional events are registered to emit a "Scene X activated" event.

I tested this on my local gateway at home throughout the last 2 days.
@chas-iot Please test this again on your setup, to verify nothing breaks.

### CODE REVIEW

**Comment by chas-iot** [comment]
I won’t get to test for a few days as I managed to trash my Zigbee installation. I’ll do a code review anyway, hopefully by tommorrow

File: zb-classifier.js [hunk 1]
```diff
@@ -2201,38 +2201,22 @@ class ZigbeeClassifier {
...
-          [`54-pressed`]: {
-            '@type': 'PressedEvent',
-            description: `Scene 2`,
-          },
-          [`104-pressed`]: {
-            '@type': 'PressedEvent',
-            description: `Scene 3`,
-          },
-          [`154-pressed`]: {
+        const eventCodes = [2, 52, 102, 153, 194, 254];
```
Comment (chas-iot) [inline zb-classifier.js:2211]: I suggest putting the scenes support in a separate function and call from here.

Comment (mrsteakhouse) [reply zb-classifier.js]: done

File: zb-classifier.js [hunk 2]
```diff
@@ -2253,10 +2253,33 @@ class ZigbeeClassifier {
             },
           });
           break;
+        case 'RC 110':
+          if (endpoint === 1) {
+            this.addSceneEvents(levelProperty, node);
+          }
```
Comment (mrstegeman) [inline zb-classifier.js:2259]: Please add a `break`

**Review by mrstegeman — CHANGES_REQUESTED** [review]

**Review by mrstegeman — APPROVED** [review]

**Pull request merged**